common/openapi.yml

```diff
@@ -691,6 +695,20 @@ components:
           format: uuid
         method_version_name:
           type: string
+    GithubMethodSourceDetailsResponse:
```

Comment:
do we still want this response here after all, given that we now have `isPrivate` in the top level of the response? (I don't think this is being used anymore after the most recent changes, but correct me if I'm wrong)